### ISSUE TITLE
No data for Turkey without county

### ISSUE THREAD

**Issue by VictorTomaili** [alpcanaydin/github-stats-for-turkey#3]
My data doesn't found in this project. Because of my county info doesn't exist. But I'm also living in Turkey.

**Comment by ogzolmz** [comment]
me too

**Comment by alpcanaydin** [comment]
Yeah there is an issue like that but all statistics are connected to a city. If you both add your city info to your locations your profile will be visible after next fetch.

**Comment by VictorTomaili** [comment]
When is next fetch? I added my city info 3-4 days ago.
